files/scripts/swss.sh

```diff
@@ -254,7 +265,10 @@ start_peer_and_dependent_services() {
             fi
         done
         for dep in ${DEPENDENT}; do
-            /bin/systemctl start ${dep}
+            state=$(is_feature_enabled $dep)
```

Comment:
@mlok-nokia, are there any dependent systemd services that are not tied to a feature? I guess most are but just wondering if you have confirmed it? CONFIG_DB hget would not return a valid state then and we will never start that service. Is that possible?

Reply:
For the case which you mentioned about, as what we know, the change is good for now. But we could have a service which it is not associated with feature table.  Since this change is to address the dhcp-relay service which has this special dependency with swss,  I will modify the code to only check for the dhcp-relay service to avoid other impact.